tutorials/auto_scheduler/tune_network_arm.py

```diff
@@ -227,6 +274,7 @@ def get_network(name, batch_size, layout="NHWC", dtype="float32"):
 
 #### TUNING OPTION ####
 network = "mobilenet"
```

Comment:
I think we need to add comment like for use_sparse =True, we should change the network to mlp, if i am not mistaken.

Reply:
I start my test with mlp, and later finding that this should work with any model.

Reply:
Okay no prob then :slightly_smiling_face: 
I agree it can work with any model which has dense layer.
I just thought you wanted to show the effect of auto-scheduler with MLP model. We can mark this comment as resolved.